scss/_variables.scss

```diff
@@ -101,6 +101,8 @@ $window-inner-padding-x: 16px !default;
 $window-inner-padding-y: 16px !default;
 $window-titlebar-padding-x: 16px !default;
 $window-titlebar-padding-y: 12px !default;
+$window-titlebar-font-size: $font-size-lg !default;
+$window-titlebar-line-height: $line-height / (unitless( $font-size-lg ) / unitless($font-size) ) !default;
```

Comment:
@gyoshev that unitless part is needed due to twbs-compat task. Bootstrap uses rem units and we use pixels. In addition, `$font-size` is not overwritten, hence we get invalid `px/rem`... Perhaps, we could add a twbs-compat-map.scss, to handle the vars that we have and bootstrap doesn't...